frontend/src/app.css

```
@@ -93,6 +96,16 @@ div.multiselect > ul.options > li.active {
 div.multiselect > ul.options > li:not(.active):hover {
 	background-color: var(--color-surface-200-800);
 }
+/* Portaled to <body>, so the rules above no longer match; theme via the library's vars. */
+ul.options.portaled-options {
+	--sms-options-bg: var(--color-surface-100-900);
+	--sms-li-active-bg: var(--color-surface-200-800);
+	--sms-options-z-index: 30;
+	color: var(--color-surface-950-50);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)